### PR TITLE
Removing lodash dependency

### LINKED ISSUE #53: Deprecate node 0.10 and 0.12
- [x] `travis.yml`: https://github.com/danethurber/webpack-manifest-plugin/blob/master/.travis.yml#L6-L7 https://github.com/danethurber/webpack-manifest-plugin/blob/master/.travis.yml#L19
- [x]  test: https://github.com/danethurber/webpack-manifest-plugin/blob/master/spec/plugin.integration.spec.js#L97-L98

### PR DESCRIPTION
**Pull request by gmac**
With this plugin gaining traction as a significant webpack/webpacker resource, seems like time to do some housekeeping that trims it down. This removes Lodash.

-----
closes #53

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "webpack-manifest-plugin",
-  "version": "1.1.2",
```
Comment (mastilver) [inline package.json]: don't update version

**Comment by mastilver** [comment]
I think this should be done as part of #53 

Let's leave it open for now, we will merge it when we are ready to release `v2`! :)

File: package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "author": "Dane Thurber <dane.thurber@gmail.com>",
   "license": "MIT",
   "engines": {
-    "node": ">=0.10"
+    "node": ">=4.3.0"
```
Comment (mastilver) [inline package.json]: `>=4`

You might also want to remove old node version from `.travis.yml`

Comment (gmac) [inline package.json]: Ah, thank you. I missed the Travis file.

**Comment by mastilver** [comment]
@gmac I already deprecated node@0.10 and node@0.12, but I didn't do any code change
So your PR is still valid and can be merged after 2.x as it won't be a breaking change (or before if you have time to work on it ;)

**Comment by andriijas** [comment]
@mastilver merge? :)

**Pull request closed without merging**